filebeat/tests/system/test_registrar.py

```diff
@@ -1346,20 +1346,16 @@ def test_registrar_meta(self):
         """
 
         self.render_config_template(
-            type='docker',
+            type='container',
             input_raw='''
-  containers:
-    path: {path}
+    paths: [{path}]
     stream: stdout
     ids:
       - container_id
-- type: docker
-  containers:
-    path: {path}
-    stream: stderr
-    ids:
-      - container_id
-            '''.format(path=os.path.abspath(self.working_dir) + "/log/")
+- type: container
```

Comment:
Should this use `container`, or `filestream` with the `container` parse?

Reply:
This should be `container` input. `filestream` input does not need meta-information about the stream. Adding a parser with the appropriate settings will let the input find the correct state in the registry.